src/pip/_internal/operations/prepare.py

```diff
@@ -376,8 +376,11 @@ def prepare_linked_requirement(
 
         # TODO: Breakup into smaller functions
         if link.scheme == 'file':
-            path = link.file_path
-            logger.info('Processing %s', display_path(path))
+            if req.from_wheel_cache:
+                logger.info('Processing %s (cached)', req.link.filename)
+            else:
+                logger.info('Processing %s', req.link.filename)
```

Comment:
If it's not from the cache, I think we should be printing the entire path (i.e. like we did before this change).